build-system/sauce_connect/start_sauce_connect.sh

```diff
@@ -22,12 +22,13 @@ YELLOW() { echo -e "\033[1;33m$1\033[0m"; }
 GREEN() { echo -e "\033[0;32m$1\033[0m"; }
 RED() { echo -e "\033[0;31m$1\033[0m"; }
 
-SC_VERSION="sc-4.5.1-linux"
+SC_VERSION="sc-4.5.3-linux"
```

Comment:
Is this a new version? We downgraded because there were problems with connecting to the service using versions `4.5.2` and `4.5.3`. See https://github.com/ampproject/amphtml/pull/20767#issue-251966287

/cc @joshmgrant

Reply:
I thought maybe somehow things settled with this on the other end, but it does look like there's still some instability here. I restared the Travis shard for a total of 6 times, 2 of them failed on non-flaky-tests reasons

Reply:
Yeah, I don't believe the underlying problem `4.5.2` and `4.5.3` is resolved yet. I think we should wait until the next version is published, and then reach out to Sauce Labs support before upgrading.